.buildkite/commands/installable-build-jetpack.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -eu
+
+# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
+echo "--- :rubygems: Fixing Ruby Setup"
+gem install bundler
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
+
+echo "--- :hammer_and_wrench: Building"
+bundle exec fastlane build_and_upload_jetpack_installable_build build_number:$BUILDKITE_PULL_REQUEST
```

Comment:
Given the similarities between the 2 scripts (WP vs JP) I wonder if it wouldn't be better to provide the target as argument?

 - Either by making a common lane `build_and_upload_installable_build app:{wordpress|jetpack} build_number:<value>` (which could then internally in the `Fastfile` switch on the parameter and call the appropriate original lane?) – and then pass down that `app:` argument from the pipeline to the `.sh` to the fastlane call
 - Or by keeping individual lanes and make the switch at the `.sh` level to call the appropriate lane (which would make a smaller change and allow to detect typos earlier)

This could even be something as simple as:
```suggestion
APP="$1"
bundle exec fastlane "build_and_upload_${APP}_installable_build" build_number:$BUILDKITE_PULL_REQUEST
```

That would avoid the duplication (and double-maintenance) for the 2 otherwise similar cases 🙂

Reply:
The repetition here exists for all of the other commands too, which was why I figured this was ok. It'd be possible to extract it all our into a shared `install-dependencies.sh` file, but in the past I've found we've run into a lot of weirdness when we try to over-DRY things 😅